runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -73,10 +73,11 @@ J9::X86::CodeGenerator::initialize()
 
    cg->setAheadOfTimeCompile(new (cg->trHeapMemory()) TR::AheadOfTimeCompile(cg));
 
-   if (!TR::Compiler->om.canGenerateArraylets() && !TR::Compiler->om.isOffHeapAllocationEnabled())
+   if (!TR::Compiler->om.canGenerateArraylets())
       {
       cg->setSupportsReferenceArrayCopy();
-      cg->setSupportsInlineStringLatin1Inflate();
+      if (!TR::Compiler->om.isOffHeapAllocationEnabled())
```

Comment:
what is the rational to keep Latin1 inflate enablement under the same clause as RefArrayCopy? i don't think Inflate uses RefArrayCopy. keeping them independent each other just looks better and more readable too.